frontend/packages/component-library/src/actionBlock/FullWidthActionBlock.tsx

```diff
@@ -0,0 +1,52 @@
+import classNames from 'classnames';
+
+import {Heading3, BodyThreeText, OverlineTwoText} from '@/typography';
+
+import {
+  ActionBlockWrapper,
+  ActionBlockProps,
+  getImage,
+  getButtons,
+  getDetail,
+} from './ActionBlock';
+
+import moduleStyles from './actionBlock.module.scss';
+
+// This component is used to render a full-width action block that's designed to be
+// used in a layout where the action block takes up the full width of its container.
+export const FullWidthActionBlock: React.FC<ActionBlockProps> = ({
+  title,
+  description,
+  image,
+  overline,
+  details,
+  primaryButton,
+  secondaryButton,
+  ...props
+}) => {
+  return (
+    <ActionBlockWrapper {...props} className={moduleStyles.fullWidth}>
```

Comment:
nit: for consistency, transparency and editability I think it might be worth updating to this:

```suggestion
  secondaryButton,
  className,
  background,
  ...HTMLAttributes
}) => {
  return (
    <ActionBlockWrapper
      background={background}
      className={classNames(moduleStyles.fullWidth, className)}
      {...HTMLAttributes}
    >
```